spec/index.adoc

```diff
@@ -3029,6 +3029,7 @@ if
 then
   S with
     canisters[M.receiver].wasm_state = res.new_state;
+    canister_version[M.receiver] = S.canister_version[M.receiver] + 1;
```

Comment:
Should the canister version increase if the method is a query method?

Reply:
Good question, and I am not sure. But probably yes: it's still a update or inter canister call; the fact that some annotation on the canister causes possible state changes to be thrown away is a detail at a different level of abstraction, inside the canister, so to say. So I assume it's more uniform to bump here too.

Reply:
In the interface spec meeting, we decided that we don't prescribe the canister version to increase after replicated query method execution. Because the canister version can also increase in other cases than those listed explicitly, it is not a breaking change to bump canister version after replicated query method execution in the future.